### PR TITLE
[JENKINS-38754] Add test coverage for Standoutput and standard error

### PR DESCRIPTION
**Pull request by imeredith**
# Description

See [JENKINS-38754](https://issues.jenkins-ci.org/browse/JENKINS-38754).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

File: src/test/js/multibranch/testResults.js [hunk 1]
```diff
@@ -40,8 +40,11 @@ module.exports = {
         blueRunDetailsPage.clickTab('tests');
 
         // Expand the test.
-        browser.useXpath().click('//div[@class="result-item-head"]');
-
-        browser.useXpath().waitForElementVisible('//div[@class="test-console"]/h4')
+        browser.useXpath().waitForElementVisible('//span[contains(text(), "StdOut")]');
```
Comment (scherler) [inline src/test/js/multibranch/testResults.js]: @imeredith please see https://github.com/jenkinsci/blueocean-acceptance-test/pull/88 and https://github.com/jenkinsci/blueocean-acceptance-test/pull/88/files#diff-fbb1a6d3d9df1b771d890e368f081658R443 

You should really drop the usage of useXpath and instead declare those paths in a page object (e.g. in runDetailsPage).

Comment (michaelneale) [inline src/test/js/multibranch/testResults.js]: as long as the assertions happen in testResults.js - I have had a nightmare of a time this week when stuff fails and it isn't directly in the test code. Takes a long time to track down. Even having the paths elsewhere is a pain as it just says @thingie failed to be found, which then takes a while to drill into when looking at a log.

**Comment by michaelneale** [comment]
LGTM

File: src/test/js/multibranch/testResults.js (hunk 1 above)
Comment (imeredith) [inline src/test/js/multibranch/testResults.js]: @scherler I disagree in this case. I think that we should add commonly used things, but this is a very specific xpath just for this test.

**Comment by michaelneale** [comment]
:bee:

**Pull request merged**